src/EFCore.MySql/Internal/MySqlRetryNoDependendiciesExecutionStrategy.cs

```diff
@@ -16,6 +19,27 @@ public class MySqlRetryNoDependendiciesExecutionStrategy
         private TimeSpan _maxRetryDelay { get; set; }
 
         /// <summary>
+        ///     The default maximum random factor, must not be lesser than 1.
+        /// </summary>
+        private const double DefaultRandomFactor = 1.1;
+
+        /// <summary>
+        ///     The default base for the exponential function used to compute the delay between retries, must be positive.
+        /// </summary>
+        private const double DefaultExponentialBase = 2;
+
+        /// <summary>
+        ///     The default coefficient for the exponential function used to compute the delay between retries, must be nonnegative.
+        /// </summary>
+        private static readonly TimeSpan _defaultCoefficient = TimeSpan.FromSeconds(1);
+
+        /// <summary>
+        ///     A pseudo-random number generator that can be used to vary the delay between retries.
+        /// </summary>
+        protected virtual Random Random { get; } = new Random();
```

Comment:
Should this be static member better? Or is this class `MySqlRetryNoDependendiciesExecutionStrategy` singleton?

Reply:
It’s not dependency injected class, it could just be private. I just copied verbatim from executionstrategy to keep the same pattern. Lmk if you want me to convert.

Good stuff though.

Reply:
It’s not dependency injected class, it could just be private. I just copied verbatim from executionstrategy to keep the same pattern. Lmk if you want me to convert.

Good stuff though.

Reply:
I don't think generate a new Random instance each time will keep the random seed, if you can, please make the random object static.

Reply:
Done.